bin/o2a-run-sys-test

```diff
@@ -671,6 +673,21 @@ function test_oozie {
     footer
 }
 
+function copy_test_artifacts {
```

Comment:
It should be "download_test_artifacts" not "copy" or "copy_test_artifacts_from_dataproc". "Copy" does not tell you what is the direction

Reply:
I updated this PR>